driver/compenv.ml

```diff
@@ -243,7 +243,6 @@ let read_one_param ppf position name v =
   | "nolabels" -> set "nolabels" [ classic ] v
   | "principal" -> set "principal"  [ principal ] v
   | "rectypes" -> set "rectypes" [ recursive_types ] v
-  | "safe-string" -> clear "safe-string" [ unsafe_string ] v
```

Comment:
could this still be supported, but ignored?

Reply:
mmh, sure. I was not sure it would make much sense for OCAMLPARAM but it seems good to keep it in.

Done.